lib/functions/host/docker.sh

```diff
@@ -572,6 +572,13 @@ function docker_cli_launch() {
 		run_host_command_logged find "${SRC}/userpatches" -name ".DS_Store" -type f -delete "||" true
 	fi
 
+	# This check is performed in order to set up the host so that it has a loop device, as calling losetup inside of
+	# docker creates a loop device but does not make it available to the already running container
+	if [[ ! -e /dev/loop0 ]]; then
+		display_alert "Running losetup in a temporary container" "because no loop devices exist" "info"
+		docker run "${DOCKER_ARGS[@]}" "${DOCKER_ARMBIAN_INITIAL_IMAGE_TAG}" /usr/sbin/losetup -f
```

Comment:
Ok, will do it this way

Reply:
Well, that breaks the docker run command! 
>"docker run" requires at least 1 argument.
>See 'docker run --help'.
>
>Usage:  docker run [OPTIONS] IMAGE [COMMAND] [ARG...]
>
>Create and run a new container from an image

Reply:
I've tried various permutations, but none of them got rid of this error. I think the `run_host_command_logged` breaks something with `DOCKER_ARGS` used which do need to be used because otherwise `losetup` can't create a device in the container. Although I could try to isolate the args that need to be used to fix that.

Reply:
Ok, `DOCKER_ARGS` was confusing `run_host_command-logged` and causing this error. I've managed to pick the minimum amount of arguments needed for it to work.